File: web3rpc/rpc-specs/code-samples/javascript/klay/configuration/forkStatus.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const forkNumber = 20;

  provider.klay
    .forkStatus(forkNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/klay/configuration/forkStatus.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

forkNumber = 20

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.fork_status(forkNumber)

print(klay_response)
